doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -284,155 +294,81 @@ Induction
       induction n.
       exact (le_n 0).
 
-.. exn:: Not an inductive product.
-   :undocumented:
-
-.. exn:: Unable to find an instance for the variables @ident ... @ident.
+.. example:: :tacn:`induction` with :n:`@occurrences`
 
-   Use in this case the variant :tacn:`elim … with` below.
+   .. coqtop:: reset all
 
-.. tacv:: induction @term as @or_and_intropattern
+      Lemma comm x y : x + y = y + x.
+      induction y in x |-   *.
+      Show 2.
 
-   This behaves as :tacn:`induction` but uses the names in
-   :n:`@or_and_intropattern` to name the variables introduced in the
-   context. The :n:`@or_and_intropattern` must typically be of the form
-   :n:`[ p` :sub:`11` :n:`... p` :sub:`1n` :n:`| ... | p`:sub:`m1` :n:`... p`:sub:`mn` :n:`]`
-   with :n:`m` being the number of constructors of the type of :n:`@term`. Each
-   variable introduced by induction in the context of the i-th goal gets its
-   name from the list :n:`p`:sub:`i1` :n:`... p`:sub:`in` in order. If there are
-   not enough names, induction invents names for the remaining variables to
-   introduce. More generally, the :n:`p`:sub:`ij` can be any
-   disjunctive/conjunctive introduction pattern (see :tacn:`intros …`). For
-   instance, for an inductive type with  one constructor, the pattern notation
-   :n:`(p`:sub:`1` :n:`, ... , p`:sub:`n` :n:`)` can be used instead of
-   :n:`[ p`:sub:`1` :n:`... p`:sub:`n` :n:`]`.
+   .. exn:: Not an inductive product.
+      :undocumented:
 
-.. tacv:: induction @term with @bindings
+   .. exn:: Unable to find an instance for the variables @ident … @ident.
 
-   This behaves like :tacn:`induction` providing explicit instances for the
-   premises of the type of :n:`term` (see :ref:`bindings`).
+      Use the variant :tacn:`elim` `… with …` instead.
```

Comment:
Should recommend using the `with @bindings` clause or the `einduction` variant (but not mention `elim`).